src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -335,10 +333,32 @@ const routes: Routes = [
     component: BlankLayoutComponent,
     children: [
       // Single Sign-On (SSO)
-      { path: 'sso/404', component: SsoNotFoundComponent },
+      {
+        path: 'sso/404',
+        component: ErrorComponent,
+        data: {
+          errorHeader: 'User Denied',
```

Comment:
How are those strings translated?
You should translate the strings here.
```suggestion
          errorHeader: this.i18n('User Denied'),
```